pkg/log/doc.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Chaos Mesh Authors.
```

Comment:
I think Readme is better than a go file

Reply:
I prefer to use godoc just nearby the code and follow the https://go.dev/doc/effective_go#commentary.

Using README might give a better experience on GitHub and local IDE, but that's the third-party tools' feature, not the language itself.